### ISSUE TITLE
logs get littered with bogus warning about "unrecognized timezone"

### ISSUE THREAD

**Issue by pkit** [ClickHouse/clickhouse-connect#181]
### Describe the bug

```
WARNING  clickhouse_connect.driver.client:client.py:124 Unrecognized timezone None received from ClickHouse
```

Any query that doesn't explicitly set `QueryResultDetails` in the `clickhouse-server` [will return with no details](https://github.com/ClickHouse/ClickHouse/blob/66847386df4f86f9bf18a90904b64f89b3b2b484/src/Server/HTTPHandler.cpp#L847).
I.e. no headers like `X-ClickHouse-Format`, `X-ClickHouse-Timezone`, `X-ClickHouse-Query-Id` would be present.

### Steps to reproduce
1. `INSERT INTO table1 VALUES (...)`
2. See the warning pop up.
3.

### Expected behaviour

No warning, as some queries are expected not to have `X-ClickHouse-Timezone` in response

### Version

Checked in the current `clickhouse-connect/main` vs `ClickHouse/master`

**Comment by genzgd** [comment]
Thanks for the report!  This should be fixed in the 0.5.23 version which will be released shortly.

**Comment by pkit** [comment]
Nice!
